.github/container/Dockerfile.jax

```diff
@@ -1,6 +1,6 @@
 # syntax=docker/dockerfile:1-labs
 ARG BASE_IMAGE=ghcr.io/nvidia/jax:base
-ARG BUILD_PATH_JAXLIB=/opt/jaxlib
+ARG BUILD_PATH_JAXLIB=/opt/jax/dist
```

Comment:
Does this need to be under `/opt/jax`?
When we use these containers for development, `/opt/jax` is sometimes mounted in from a (slow) network filesystem, so keeping as many build artifacts as possible off it can be beneficial.

Reply:
oh I see, engineers could mount their own `jax` devel repo to `/opt/jax` for building.